frontend/src/views/rescue/List.vue

```diff
@@ -254,7 +252,7 @@ const filteredRescue = computed(() => {
 })
 
 const showRescues = computed(() => {
-	return rescues.data && !rescues.loading
+	return rescues.data.length && !rescues.loading
```

Comment:
What if data is undefined? Which is the case when data isn't loaded yet, or some error occurred. 

You can do this instead:

```suggestion
	return rescues.data?.length && !rescues.loading
```

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Thank you, will make the recommended changes